pipeline/metadata/6.1.yaml

```diff
@@ -726,6 +726,17 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "test-rgw-ms-deployment-via-Haproxy-with-archive"
+          execution_time: "2h 29m 53s"
+          suite: "suites/pacific/rgw/tier-3_rgw_multisite_archive_with_haproxy.yaml"
+          global-conf: "conf/pacific/rgw/tier-3_rgw_multisite_archive_with_haproxy_conf.yaml"
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - rgw
+
```

Comment:
Please update the metadata of 5.3 and 6.0, since suite is in pacific and part of 6.2 metadata file

Reply:
@anrao19 Done.